boot/office/src/main/resources/application.yml

```
@@ -14,3 +14,6 @@ spring:
     login:
       max-fail-count: 10
       block-duration-seconds: 3600 # 1 시간
+
+toss:
+  secret-key: ${TOSS_KEY}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)